### DISCUSSION TITLE
Longer flows than 7 minutes on Maestro Cloud

### DISCUSSION THREAD

**Discussion by jpc879** [mobile-dev-inc/Maestro#1712] [General]
Hello,
is there a way to run flows on Maestro Cloud that are longer than 7 minutes before the timeout? We have an app that has test cases that would take longer than that, and splitting it into separate flows would not be possible.
I'd appreciate any help.

**Comment by simon-gilmurray** [comment]
Also would appreciate a bump to 10 minutes timeout on Cloud
